package.json

```diff
@@ -160,7 +160,7 @@
     "react-native-picker-select": "git+https://github.com/Expensify/react-native-picker-select.git#da50d2c5c54e268499047f9cc98b8df4196c1ddf",
     "react-native-plaid-link-sdk": "11.11.0",
     "react-native-qrcode-svg": "6.3.11",
-    "react-native-quick-sqlite": "git+https://github.com/margelo/react-native-quick-sqlite#99f34ebefa91698945f3ed26622e002bd79489e0",
+    "react-native-quick-sqlite": "git+https://github.com/margelo/react-native-nitro-sqlite#99f34ebefa91698945f3ed26622e002bd79489e0",
```

Comment:
NAB:

```suggestion
    "react-native-nitro-sqlite": "git+https://github.com/margelo/react-native-nitro-sqlite#99f34ebefa91698945f3ed26622e002bd79489e0",
```

Reply:
I mean that we change the name of dependency for consistency with the name of package

Reply:
Yeah, should we let's update the dependency name to align with the new name, too?

Reply:
Ahhh, Got it

Reply:
We use `react-native-quick-sqlite` package with another branch

Reply:
@shubham1206agra we should include some explanation in the PR description to make it clearer for other ones